data/tree/node setter (mutable)/ec47f31f-a2f2-4f53-ab3e-a5345e5bca41.json

```diff
@@ -10,18 +10,19 @@
       "1670d7c3-36ce-42d6-8411-b90866ca411a": {
         "type": "(string or number)[]",
         "name": "path",
-        "description": "Specifies the location of the node."
+        "description": "Receives the path segments where the `node` should  be written to"
       },
       "5436f8ba-84e3-48e2-bd39-9ccaa8aeea9a": {
         "type": "any",
         "name": "node",
-        "description": "The node to be stored in the tree."
+        "description": "Receives the node to be written"
```

Comment:
Maybe use the wording "to be set" as it is more conform to the name of the node and its description.